src/utils/constants.js

```diff
@@ -0,0 +1,16 @@
+import EnvLib from '@adobe/aio-lib-env'
+
+const { getCliEnv } = EnvLib
+const clientEnv = getCliEnv()
+
+const PROD_CONSTANTS = {
+  AIO_CLI_API_KEY: 'aio-cli-console-auth',
```

Comment:
These are the keys that the `aio console` uses. We have been using these in API Mesh for a while now